### PR TITLE
Display all relations

### PR DESCRIPTION
**Pull request by tmushayahama**
### Issues 

- https://github.com/geneontology/wc-gocam-viz/issues/49
- https://github.com/geneontology/wc-gocam-viz/issues/38
- https://github.com/geneontology/wc-gocam-viz/issues/40
- https://github.com/geneontology/wc-gocam-viz/issues/33
- https://github.com/geneontology/wc-gocam-viz/issues/37
- https://github.com/geneontology/wc-gocam-viz/issues/41
- https://github.com/geneontology/wc-gocam-viz/issues/50

Changed

- Added more relations indirect regulates, constitutively, etc
- use of proper gylps and edge lines
- fixed legend
- not showing disallowed Relations
- cleaning up unused code

### In Progresss

- removes input for 
- is small molecule inhibitor relation
- code cleanup, so much unused code
- DB XREF code calling GitHub raw (@kltm, @pkalita-lbl this fails sometimes and no a good idea to use github as server

@pkalita-lbl are you using show-popup property, I kinda have deleted it. I am not sure if its is unused code

cc @vanaukenk @thomaspd @dustine32

### CODE REVIEW

**Comment by tmushayahama** [comment]
Also in preparation for you combine repo @pkalita-lbl we can discuss about some questionable dependences here. We can find better alternatives or inhouse. I have added moment, but it seems it is not favored. If you have an alternative  And also dbxref idk what it is doing it has like one function etc

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "cytoscape-dagre": "^2.4.0",
     "graphlib": "^2.1.8",
     "lodash": "^4.17.21",
+    "moment": "^2.30.1",
```
Comment (pkalita-lbl) [inline package.json:34]: You probably already know that Moment has their own recommendations for alternatives: https://momentjs.com/docs/#/-project-status/recommendations/

Personally I've used [date-fns](https://date-fns.org/) and have had no issues with it. But I think that's something that can be addressed separately from this PR.

File: src/styled.html [hunk 2]
```diff
@@ -6,62 +6,58 @@
...
 
 <body>
 
   <div style='max-width: 1200px; margin: 0 auto'>
-    <wc-gocam-viz
-      id='gocam-1'
-      gocam-id='568b0f9600000284'
-      show-legend='true'
-    >
+    <wc-gocam-viz id='gocam-1' api-url='http://barista-dev.berkeleybop.org/search//stored?id=%ID'
```
Comment (pkalita-lbl) [inline src/styled.html:59]: Setting the `api-url` attribute here is fine locally if that's needed for your testing, but that change shouldn't be committed.

**Review by pkalita-lbl — CHANGES_REQUESTED** [review]
My immediate concern is about the use of `api-url` in `src/styled.html`.

Everything else looks fine... _I think_?? Am I right in understanding that a lot of this code is copy and pasted (and maybe lightly modified?) from here https://github.com/geneontology/noctua-form-base/tree/master/src/%40noctua.form? 

If that's the case we need to figure out a better way to make the shared code accessible to whatever clients need it. To be clear I'm not saying to do that as part of this PR, and maybe it's even a separate project. But the current situation seems somewhat challenging to maintain.

File: src/styled.html (hunk 2 above)
Comment (tmushayahama) [reply src/styled.html]: yes, I have reverted the styles.html

File: package.json (hunk 1 above)
Comment (tmushayahama) [reply package.json:34]: I am having problems with date-fns and typescript, 'Intl' has no exported member named 'RelativeTimeFormatOptions'. Did you mean 'DateTimeFormatOptions'? so can I look into this later then I remove moment later. or moment has to be removed before PR. I had changed the date to use date-fns and working but that tpescript error

**Comment by tmushayahama** [comment]
@pkalita-lbl Yes, most of the code is from noctua form base VPE and I am planning to fix that if they agree to continue this route. It seemed there was a new proposal of using NDEx or some other lib. So I didn't want to spend time trying to make it accessible if it is temporary solution. However, I once did have the npm and then https://www.npmjs.com/package/@geneontology/noctua-form-base

But refactoring more, maybe only few functions are needed because this is a readonly and noctua-form-base is for the whole CRUD

File: package.json (hunk 1 above)
Comment (pkalita-lbl) [reply package.json:34]: Without knowing what you tried doing, the error message in isolation isn't too informative. I'm not even really strongly advocating for `date-fns`. I was just pointing out that I've used it (yes, in TypeScript projects) and thought it was fine.

But more importantly, to quote myself 😂 :

> I think that's something that can be addressed separately from this PR.

**Comment by pkalita-lbl** [comment]
> It seemed there was a new proposal of using NDEx or some other lib.

That's long, long, long-term thinking. I wouldn't even call it a proposal or a plan. It's a glimmer of an idea that may or may not happen. If it does happen it won't be for a while.

> However, I once did have the npm

I think that's more along the lines of what I was imagining: moving the shared pieces to a separate NPM package that can be used by the VPE and the widget. Maybe some day we can chat with @kltm about how much work that would be and how to prioritize it among other work.

**Review by pkalita-lbl — APPROVED** [review]

**Pull request merged**